.cbmc-batch/jobs/Makefile.common

```diff
@@ -23,6 +23,12 @@ sinclude Makefile.local
 SPACE :=$() $()
 COMMA :=,
 
+################################################################
+# Useful macros
+C_FROM_GOTO = $(patsubst $(GOTODIR)%, $(SRCDIR)%, $(1:.goto=.c))
+GOTO_FROM_C = $(patsubst $(SRCDIR)%, $(GOTODIR)%, $(1:.c=.goto))
+LOG_FROM_GOTO = $(patsubst $(GOTODIR)%, $(LOGDIR)%, $(1:.goto=.log))
```

Comment:
I always thought that argument separators have to be just a comma (with no space afterwards) in Make, and the GNU Make manual itself says that. However, the actual behaviour seems to be that Make strips all whitespace before/after a comma, so this code does work.

I would have written this as
```Make
C_FROM_GOTO = $(patsubst $(GOTODIR)%,$(SRCDIR)%,$(1:.goto=.c))
```
which is less readable. But I don't have strong feelings about this, since your version does work.

Reply:
Done